<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientManagerForm</class>
 <widget class="QWidget" name="ClientManagerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client Info</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="listVerticalLayout">
       <item>
        <widget class="QTableView" name="tableView">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>true</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="textElideMode">
          <enum>Qt::ElideRight</enum>
         </property>
         <property name="showGrid">
          <bool>true</bool>
         </property>
         <property name="gridStyle">
          <enum>Qt::NoPen</enum>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>40</number>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>90</number>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QToolBox" name="toolBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="inputPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>384</width>
         <height>288</height>
        </rect>
       </property>
       <attribute name="label">
        <string>&amp;Input</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QFormLayout" name="clientFormLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="idLabel">
            <property name="text">
             <string>I&amp;D</string>
            </property>
            <property name="buddy">
             <cstring>idLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="idLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="nameLabel">
            <property name="text">
             <string>Name</string>
            </property>
            <property name="buddy">
             <cstring>nameLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="nameLineEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="genderLabel">
            <property name="text">
             <string>Gender</string>
            </property>
            <property name="buddy">
             <cstring>genderLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="genderLineEdit"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="ageLabel">
            <property name="text">
             <string>Age</string>
            </property>
            <property name="buddy">
             <cstring>ageLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="ageLineEdit"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="phoneNumberLabel">
            <property name="text">
             <string>Phone Number</string>
            </property>
            <property name="buddy">
             <cstring>phoneNumberLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="phoneNumberLineEdit"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="addressLabel">
            <property name="text">
             <string>Address</string>
            </property>
            <property name="buddy">
             <cstring>addressLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="addressLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="clearPushButton">
          <property name="text">
           <string>&amp;Clear</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="addHorizontalLayout">
          <item>
           <widget class="QPushButton" name="addPushButton">
            <property name="whatsThis">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/i18n/CSApp_ko_KR.qm&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>&amp;Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="modifyPushButton">
            <property name="text">
             <string>&amp;Modify</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="searchPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>384</width>
         <height>288</height>
        </rect>
       </property>
       <attribute name="label">
        <string>&amp;Search</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTableView" name="searchTableView">
            <property name="toolTipDuration">
             <number>-1</number>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="showGrid">
             <bool>false</bool>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QComboBox" name="searchComboBox">
              <item>
               <property name="text">
                <string>ID</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Name</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Gender</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Age</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Phone Number</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Address</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="searchLineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="searchPushButton">
            <property name="text">
             <string>S&amp;earch</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>idLineEdit</tabstop>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>genderLineEdit</tabstop>
  <tabstop>ageLineEdit</tabstop>
  <tabstop>phoneNumberLineEdit</tabstop>
  <tabstop>addressLineEdit</tabstop>
  <tabstop>clearPushButton</tabstop>
  <tabstop>addPushButton</tabstop>
  <tabstop>modifyPushButton</tabstop>
  <tabstop>searchComboBox</tabstop>
  <tabstop>searchLineEdit</tabstop>
  <tabstop>searchPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
